static/src/javascripts/projects/common/modules/ui/dropdowns.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import bean from 'bean';
+import fastdom from 'fastdom';
+
+const containerSelector = '.dropdown';
+const buttonCN = 'dropdown__button';
+const contentCN = 'dropdown__content';
+
+const updateAria = (container: Element): void => {
+    const v: boolean = container.classList.contains('dropdown--active');
+    const content = [...container.getElementsByClassName(contentCN)];
+    const button = container.getElementsByClassName(buttonCN);
+
+    content.forEach((c: Element) => {
+        c.setAttribute('aria-hidden', v ? 'false' : 'true');
```

Comment:
Good call! What happens is that the boolean is coerced into a string before the call to `setAttribute`. To keep Flow happy, I'm doing this explicitly.